app/models/user.rb

```diff
@@ -121,19 +121,6 @@ def can_access_id_card?
     false
   end
 
-  def self.from_merged_attrs(existing_user, new_user)
-    # we want to always use the more recent attrs so long as they exist
-    attrs = new_user.attributes.map do |key, val|
-      { key => val.presence || existing_user[key] }
-    end.reduce({}, :merge)
-
-    # for loa, we want the higher of the two
-    attrs[:loa][:current] = [existing_user[:loa][:current], new_user[:loa][:current]].max
-    attrs[:loa][:highest] = [existing_user[:loa][:highest], new_user[:loa][:highest]].max
```

Comment:
This code was in place because of bad data from ID.me where up-level'ing would return an `loa.highest=nil`.  So long as that has been fixed up, it's fine to remove this.